android-json-form-wizard/src/main/java/com/vijay/jsonwizard/interfaces/JsonApi.java

```diff
@@ -119,4 +120,10 @@ void addOnActivityRequestPermissionResultListener(Integer requestCode,
     boolean isPreviousPressed();
 
     void setPreviousPressed(boolean previousPressed);
+
+    void makeFieldHidden(JSONObject fieldObject);
+
+    void makeFieldDisabled(JSONObject fieldObject);
```

Comment:
This are not required since their implementations are returning values of `public static` methods in a utils class